modules/extensions-spring-support/src/main/java/org/mule/module/extension/internal/config/ConnectionProviderBeanDefinitionParser.java

```diff
@@ -49,6 +52,9 @@ public ConnectionProviderBeanDefinitionParser(ConnectionProviderModel providerMo
     @Override
     protected void doParse(BeanDefinitionBuilder builder, Element element, ParserContext parserContext)
     {
+        boolean isPoolingProfileAlreadyDefined = false;
```

Comment:
why are these two needed? It shouldn't be possible to define any of those elements in the XSD